taxcalc/growdiff.json

```diff
@@ -0,0 +1,205 @@
+{
+    "_ABOOK": {
+        "long_name": "ABOOK additive difference from default projection",
+        "description": "Default projection is in growfactors.csv file.  ABOOK extrapolates Records variables: e07300 and e07400.",
```

Comment:
Yes, changes will be rare.  And when making any change the `growdiff.json` file will have to be edited, so the need to revise the "description" contents will be obvious.

@MattHJensen @andersonfrailey